core/cmd/remote_client_test.go

```diff
@@ -1323,32 +1322,43 @@ func TestClient_SetLogConfig(t *testing.T) {
 	}
 	client, _ := app.NewClientAndRenderer()
 	client.CookieAuthenticator = cmd.NewSessionCookieAuthenticator(app.Config.Config, &cmd.MemoryCookieStore{})
-	client.HTTP = cmd.NewAuthenticatedHTTPClient(config, client.CookieAuthenticator, sr)
+	client.HTTP = cmd.NewAuthenticatedHTTPClient(app.Config, client.CookieAuthenticator, sr)
```

Comment:
You don't need to set your own `CookieAuthenticator` or `HTTP` on the `client` as the `app.NewClientAndRenderer` already returns you a mocked logged in client. 

The `AutoLogin` test needs this because it needs to to test a login.

So like most of the tests you should only need to setup with:

```
app := startNewApplication(t)
client, _ := app.NewClientAndRenderer()
```

Then you can remove all the other changes

Reply:
Roger makes sense, looks now to be included in: https://github.com/smartcontractkit/chainlink/pull/4124

Will close this out~